notebooks/290-magika-content-type-recognition/290-magika-content-type-recognition.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        return {result.output.ct_label: float(result.output.score) for result in results}
</pre><p>Looks like we should choose another output format. For example, we can have 5 different results and due to PredictionMode.HIGH_CONFIDENCE and scores low then thresholds we can get 5 unknown. In that case we will show only the last result. Probably <code>return {result.dl.ct_label: float(result.output.score) for result in results}</code> and separately final verdict. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1826/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>